tests/steps/smoke_steps.py

```diff
@@ -213,7 +216,35 @@ def step_impl(context, version):
 
     # compare the version with the expected one
     eq_(returned_version, version,
-        'Not the correct version: found({}) expected({})'.format(returned_version, version))
+        '[{}] Not the correct version: found({}) expected({})'.format(comp, returned_version, version))
+
+    __logger__.debug("{} Version: {}".format(comp, returned_version))
+
+
+@then(u'the returned version from "CYGNUS" should match the "(?P<version>.+)"')
+def step_impl(context, version):
+    comp = "CYGNUS"
+
+    # Cast version returned (if any)
+    returned_version = component_version_check(context, component=comp)
+
+    # compare the version with the expected one
+    eq_(returned_version, version,
+        '[{}] Not the correct version: found({}) expected({})'.format(comp, returned_version, version))
+
+    __logger__.debug("{} Version: {}".format(comp, returned_version))
+
+
+@then(u'the returned version from "PEP" should match the "(?P<version>.+)"')
```

Comment:
The platform components returns the value of the version in a different formats that require a special treatment to extract it so we need the different version steps. But you are right that the Node components version checkers do the same... unified 

Fixed in 84fb2d8